### PR TITLE
Dragging for links fixed (closes #2529)

### LINKED ISSUE #2529: The Drag action works incorrectly
### Are you requesting a feature or reporting a bug?

Bug.

### What is the current behavior?

An element does not move or moves a lot less than specified in action. 

### How would you reproduce the current behavior (if this is a bug)?

Run the specified test.

#### Provide the test code and the tested page URL (if applicable)

Tested page URL:
https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_iframe
Test code

```js
import { Selector } from 'testcafe';

fixture `New Fixture`
    .page `https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_iframe`;

test('New Test', async t => {
    await t
        .drag(Selector('#dragbar'), -600, 0)
        .expect(Selector('#dragbar').offsetLeft).lt(500);
});
```

### Specify your

* operating system: Windows 10
* testcafe version: 0.20.3
* node.js version: 8.11.1

### PR DESCRIPTION
**Pull request by helen-dikareva**

### CODE REVIEW

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

**Comment by helen-dikareva** [comment]
please review

File: src/client/automation/playback/move/index.js [hunk 1]
```diff
@@ -426,7 +429,8 @@ export default class MoveAutomation {
 
                 var draggable = findDraggableElement(this.dragElement);
 
-                if (draggable && featureDetection.hasDataTransfer) {
+                // NOTE: we should slip simulation default drag&drop behavior if `mousedown` event was prevented (GH - 2529)
```
Comment (churkin) [inline src/client/automation/playback/move/index.js]: `slip`

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

**Comment by AndreyBelym** [comment]
@testcafe-build-bot retest

File: test/functional/fixtures/api/es-next/drag/test.js [hunk 2]
```diff
@@ -139,5 +139,9 @@ describe('[API] Drag actions', function () {
         it('Should reproduce native browser behavior', function () {
             return runTests('./testcafe-fixtures/drag-and-drop-test.js', 'drag link and image', { only: ['chrome'] });
         });
+
+        it('Should not raise drag and drop events if mousedown event was prevents', function () {
```
Comment (churkin) [inline test/functional/fixtures/api/es-next/drag/test.js]: `prevents` --> 'prevented'

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

File: src/client/automation/playback/move/index.js [hunk 3]
```diff
@@ -426,7 +429,8 @@ export default class MoveAutomation {
 
                 var draggable = findDraggableElement(this.dragElement);
 
-                if (draggable && featureDetection.hasDataTransfer) {
+                // NOTE: we should skip simulation default drag&drop behavior if `mousedown` event was prevented (GH - 2529)
```
[the remark this answers is not in the document]
Comment (helen-dikareva) [reply src/client/automation/playback/move/index.js]: You've already approved it yesterday) 
If you want to make notes in the same style create a separate PR.

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

**Comment by helen-dikareva** [comment]
Finally WPR

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

**Comment by churkin** [comment]
@testcafe-build-bot \retest

**Comment by helen-dikareva** [comment]
@testcafe-build-bot \retest

**Comment by AndreyBelym** [comment]
@testcafe-build-bot retest

**Comment by AndreyBelym** [comment]
We have a problem with client tests in Safari 11 on Saucelabs. It doesn't introduced by this PR, because it can be reproduced in other branches.

**Pull request merged**